src/theory/quantifiers/sygus/sygus_grammar_cons.cpp

```diff
@@ -1175,14 +1175,16 @@ void CegGrammarConstructor::mkSygusDefaultGrammar(
     {
       //add for testers
       Trace("sygus-grammar-def") << "...add for testers" << std::endl;
-      const Datatype& dt = types[i].getDatatype();
+      const DType& dt = types[i].getDType();
       std::vector<TypeNode> cargsTester;
       cargsTester.push_back(unres_types[iuse]);
       for (unsigned k = 0, size_k = dt.getNumConstructors(); k < size_k; ++k)
       {
-        Trace("sygus-grammar-def") << "...for " << dt[k].getTesterName() << std::endl;
-        sdtBool.addConstructor(
-            dt[k].getTester(), dt[k].getTesterName(), cargsTester);
+        Trace("sygus-grammar-def")
+            << "...for " << dt[k].getTester() << std::endl;
+        std::stringstream sst;
+        sst << dt[k].getTester();
```

Comment:
This is a bit ugly. Maybe we should also have `DTypeConstructor::getTesterName()`?

Reply:
Tester names are a user-level piece of information to support deprecated (pre SMT-LIB 2.6) custom tester names. We removed this field from the Node-level constructor, I would not want to readd this.

Reply:
Hmm, that's fair but how is this going to work when we remove the `Expr` layer? Who will maintain that information and how do we access it?

Reply:
I think we should remove that field.  Testers no longer have indepdent names in SMT-LIB, instead their syntax is e.g. `(_ is cons)`.

Reply:
That sounds good to me.